tests/test_io.py

```diff
@@ -65,13 +51,14 @@ def test_can_crop_waveform():
 
 def test_crops_are_correct_shape():
     sr = 160001
-    secs = random.randint(5, 11)
-    waveform = torch.randn(1, secs * sr)
-    loader = Audio()
-    shape = None
-    for segment in SlidingWindow(end=secs):
-        wav, sr = loader.crop({"waveform": waveform, "sample_rate": sr}, segment)
-        if shape is None:
-            shape = wav.shape
-        else:
-            assert shape == wav.shape
+    for secs in range(5, 11):
+        waveform = torch.randn(1, secs * sr)
+        loader = Audio()
+        shape = None
+        print(secs - 0.01)
+        for segment in SlidingWindow(end=secs - 2 * 0.01):
+            wav, sr = loader.crop({"waveform": waveform, "sample_rate": sr}, segment)
```

Comment:
If the point of this test is to check that `Audio.crop` always returns the same number of samples for the same input segment length -- then you are missing a `fixed` kwargs in `Audio.crop`, whose whole point is to be robust to otherwise random rounding errors.

Reply:
Perfect, I'll change this then, missed that sorry.